### PR TITLE
Development gui

### PR DESCRIPTION
**Pull request by Tymoteuszauke**
Updated README.md (includes how to run client application from terminal using maven).
Added AppClient with main method.

### CODE REVIEW

File: client/src/main/java/com/blackship/battlesheep/AppClient.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+/**
+ * @author Mateusz Słaboński
+ * @since 27.07.17
+ */
+public class AppClient  {
+    public static void main(String[] args) {
+        new Thread(() -> {
+           Application.launch(ConnectController.class);
+        }).start();
+    }
```
Comment (piotr921) [inline client/src/main/java/com/blackship/battlesheep/AppClient.java:15]: can be replaced with (Intellij advice): 
new Thread(() -> Application.launch(ConnectController.class)).start();

File: client/src/main/java/com/blackship/battlesheep/fx/controllers/BoardController.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import javafx.scene.control.TextField;
+import javafx.scene.layout.GridPane;
+
+/**
+ * @author Mateusz Słaboński on 27.07.17
+ * @project
+ */
+public class BoardController {
+
+    private final static int BOARD_SIZE = 10;
```
Comment (piotr921) [inline client/src/main/java/com/blackship/battlesheep/fx/controllers/BoardController.java:19]: private :+1:

File: client/src/test/java/com/blackship/battlesheep/fx/controllers/utils/PositionUtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
+package com.blackship.battlesheep.fx.controllers.utils;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
```
Comment (piotr921) [inline client/src/test/java/com/blackship/battlesheep/fx/controllers/utils/PositionUtilsTest.java:6]: Import '*'

File: client/src/test/java/com/blackship/battlesheep/fx/controllers/utils/PositionUtilsTest.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+/**
+ * @author Mateusz Słaboński on 27.07.17
+ * @project
+ */
+public class PositionUtilsTest {
+
+    public static final int BOARD_SIZE = 10;
```
Comment (piotr921) [inline client/src/test/java/com/blackship/battlesheep/fx/controllers/utils/PositionUtilsTest.java:14]: change public to private

**Review by piotr921 — APPROVED** [review]

**Review by arturAR — APPROVED** [review]

**Pull request merged**